### PR TITLE
Various railjson generator fixes

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

File: python/railjson_generator/railjson_generator/infra_builder.py [hunk 1]
```diff
@@ -30,57 +30,30 @@ def add_track_section(self, *args, **kwargs):
...
-        right: TrackEndpoint,
-        signal_on_ports: Mapping[str, Tuple[str, str]] = None,
-        **kwargs
-    ):
+    def add_point_switch(self, base: TrackEndpoint, left: TrackEndpoint, right: TrackEndpoint, **kwargs):
         """
         Adds a switch as well as all links between concerned track sections.
         """
         switch = PointSwitch(base=base, left=left, right=right, **kwargs)
-        l1 = self.add_link(base, left)
```
Comment (Castavo) [inline python/railjson_generator/railjson_generator/infra_builder.py:45]: This function created a link but also registered the two endpoint as neighbors
Now they no longer are neighbors so railjson-generator's pathfinding algorithm will be broken I believe
Is this intended ?

**Review by Castavo — COMMENTED** [review]
Nice cleaning up of the generator :raised_hands:

File: python/railjson_generator/railjson_generator/infra_builder.py (hunk 1 above)
Comment (multun) [reply python/railjson_generator/railjson_generator/infra_builder.py:45]: it wasn't, thanks :)

**Review by flomonster — APPROVED** [review]
Can you bump the version of the library?

**Pull request merged**